test/util/generator_helpers.rb

```diff
@@ -146,5 +144,15 @@ def generate
                        fixture_info['options'],
                        fixture_info['spec'])
     end
+
+    # Validates multi-platform specs
+    get_platforms(0).zip(
```

Comment:
Nice! Just curious, for the same set of osinfo data does this always produce the same set of fixtures? I ask mostly because I'm not sure about the `shuffle` method...